### PR TITLE
Date time picker additional time zone info

### LINKED ISSUE #455: DST button in date-time-picker needs more descriptive text on CDT / CST button
# Bug Report

## Description
When selecting a time that falls in the transition from Daylight Savings Time to Standard Time, a modal is presented. After selecting a time, a button appears after the time input that will either read, "CDT" or "CST".  This abbreviation may be unclear to users.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-core/#/tests/terra-date-time-picker/date-time-picker/date-time-picker-dst
2. Set time to 2:30
3. Select Before or After button in modal popup.
4. Shift focus to CDT/CST button

## Expected Behavior
Button should communicate the abbreviate button content to users.

## Possible Solution
Add aria-label to button to communicate the abbreviated content.

## Environment

* Component Name and Version: terra-date-time-picker
* Browser Name and Version: Mobile Safari w VoiceOver on iOS 12

### PR DESCRIPTION
**Pull request by jmsv6d**
### Summary
Fixes #455 Adds title and aria-label with timezone name, abbreviation, and UTC offset to the daylight savings/standard time button.

### Deployment Link
https://terra-framework-deploye-pr-687.herokuapp.com/#/components/terra-date-time-picker/date-time-picker/date-time-picker

### CODE REVIEW

File: packages/terra-date-time-picker/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated daylight savings time/standard time button to have aria-label and title with additional timezone info, to clarify it's meaning.
```
Comment (benbcai) [inline packages/terra-date-time-picker/CHANGELOG.md]: it's -> its

**Comment by mmalaker** [comment]
Anyone know what's up with this https://terra-framework-deploye-pr-687.herokuapp.com/?
Screen Shot 2019-06-03 at 3 45 01 PM

**Comment by mjhenkes** [comment]
@mmalaker this change should fix the issue
https://github.com/cerner/terra-framework/pull/707

**Comment by neilpfeiffer** [comment]
Tested: PR #687
 
**Reviewed:**
- Adds additional tooltip and aria-label with best description available: the combination of the zone name, timezone abbreviation, & GMT offset – since the long format of the timezone abbreviation is not provided by Moment.
 
 
Verified by @neilpfeiffer, Ready for Release.
 
**Pass** |
-----------|
UX Review|

**Comment by mmalaker** [comment]
One issue noted during functional verification.  On mobile, the meridiem (a.m./p.m.) toggle is cut off and not reachable on the example site. Probably just a site issue and not a component issue.

**Comment by neilpfeiffer** [comment]
@mmalaker Actually not a dev site example only issue - was a found bug and reported here:
https://github.com/cerner/terra-framework/issues/706

**Comment by mmalaker** [comment]
Functional verification completed. The date time picker examples display and function as expected with the exception of the issue #706 above.

**Pull request merged**